index.php

```diff
@@ -81,25 +81,34 @@ function getLastRevision($distro) {
 
 To install the latest stable version:
 <pre>
-bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
+sudo -E bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
```

Comment:
I don't think the sudo -E is still necessary, no?

Reply:
Oh I inherited @ZenithalHourlyRate's documentation without checking. Now the environment variable should not be needed to pass by .